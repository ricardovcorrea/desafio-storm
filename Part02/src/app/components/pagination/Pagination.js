import React from "react";
import "./Pagination.scss";

import Button from "@material-ui/core/Button";
import CustomButtom from "../button/Button";

const Pagination = () => {
  return (
    <div className="pagination">
      <Button
        disableElevation
        variant="contained"
        className="paginationButtonInactive"
      >
        Primeiro
      </Button>
      <Button
        disableElevation
        variant="contained"
        className="paginationButtonInactive"
      >
        Anterior
      </Button>
      <CustomButtom bgColor="#D83367" color="white" text="1" />
      <Button
        disableElevation
        variant="contained"
        className="paginationButtonInactive"
      >
        Próximo
      </Button>
      <Button
        disableElevation
        variant="contained"
        className="paginationButtonInactive"
      >
        Último
      </Button>
    </div>
  );
};

export default Pagination;
