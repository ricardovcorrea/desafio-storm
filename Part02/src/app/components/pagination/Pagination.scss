.pagination {
  width: 100%;
  justify-content: center;
  display: flex;
  margin-top: 30px;
}

.paginationButtonInactive {
  background-color: #fafafa !important;
  text-transform: none !important;
  margin: 5px !important;
  font-style: normal !important;
  color: #d1cece !important;
}
