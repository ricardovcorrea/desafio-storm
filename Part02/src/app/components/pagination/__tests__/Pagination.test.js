import React from 'react';
import { render } from '@testing-library/react';
import Pagination from '../Pagination';

test('renders pagination', () => {
  const container = render(<Pagination />)
  expect(container.container).toMatchSnapshot()
});
