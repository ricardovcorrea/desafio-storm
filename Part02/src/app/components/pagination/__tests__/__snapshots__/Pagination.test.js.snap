// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pagination 1`] = `
<div>
  <div
    class="pagination"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained paginationButtonInactive MuiButton-disableElevation"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Primeiro
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained paginationButtonInactive MuiButton-disableElevation"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Anterior
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained button"
      style="background-color: rgb(216, 51, 103);"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        <span
          style="color: white;"
        >
          1
        </span>
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained paginationButtonInactive MuiButton-disableElevation"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Próximo
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained paginationButtonInactive MuiButton-disableElevation"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        Último
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
